packages/store/src/execution/dispatchOutsideZoneNgxsExecutionStrategy.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  NgZone,
+  PLATFORM_ID,
+  Injectable,
+  ɵNoopNgZone as NoopNgZone,
+  Inject
+} from '@angular/core';
+import { isPlatformServer } from '@angular/common';
+
+import { NgxsExecutionStrategy } from './symbols';
+import { FunctionExpression } from 'typescript';
```

Comment:
FunctionExpression - where usage?

Reply:
Argh! Must have been added when I was exploring the typescript types.
I'll remove.